src/lib/notificationManager.ts

```
@@ -151,30 +152,7 @@ class NotificationManager {
         transparent: true
       })
 
-      webviewWindow.once('tauri://load-start', () => {
```

is this section not needed anymore?

---

Very strange! Was it trying to create the same notification multiple times?

---

I'm not sure. Looking at Sentry it seems like this error has been registering since the very beginning

---

Reason I bring it up is, things failing silently usually isn't better unless we know why it's failing and we can make some kind of note of that in the code. 

My recommendation here is:
- keep the code until we understand it
- in the catch statement, don't log out the error if it matches this error that's a false negative

This way we can still look into what is going on. If this false negative error stiuation isn't a reported issue with the plugin, it probably means we are doing something wrong.

---

With this, we can merge in your updates without having to solve the problem right away, and we still ahve the code so we won't just forget about it forever

---

Smart.

Okay I just reverted the changes. I made it still log the errors, but not toast them. Just the same as it was before. Is that good?

---

Yes! Thanks for humoring me :) It should help us long term

---

Agreed, which by the way I think I already figured out the root problem: the FlowPage component is being rendered like 10+ times when a session starts. The webview creation is failing because it can only create one type of notification at a time. This is prob also why the Shopify player shuffles a few times when you start a session.

---

Nice work! That makes sense. Probably good to get that sorted out. I had noticed that with starting spotify as well